api/proto/external.proto

```diff
@@ -53,6 +53,32 @@ message Ed25519Signature {
     bytes data = 1;
 }
 
+// Complete AccountKey, containing the pair of secret keys, which can be used
+// for spending, and optionally some Fog related info that can be used for spending.
+//
+// This matches the Rust `transaction::AccountKey` struct.
+message AccountKey {
+    // Private key 'a' used for view-key matching.
+    RistrettoPrivate view_private_key = 1;
+
+    // Private key `b` used for spending.
+    RistrettoPrivate spend_private_key = 2;
+
+    // Optional url of fog server. Empty string when not in use.
+    string fog_url = 3;
+}
+
+// A public address, used to identify recipients.
+message PublicAddress {
```

Comment:
`mc_transaction_core::account_keys::PublicAddress` should also be made serializable with prost. (Another PR is fine though)